### ISSUE TITLE
explain what optax.centralize does

### ISSUE THREAD

**Issue by fabianp** [google-deepmind/optax#1080]
Its docstring is extremely poor: https://optax.readthedocs.io/en/latest/api/transformations.html#optax.centralize

  * Explain what centralize means
  * Add a usage example
  * Put reference in correct RST format

**Comment by miguelcsx** [comment]
Hi @fabianp , I'm going to start working on this issue, could you assign it to me?

**Comment by fabianp** [comment]
thanks for picking this up @miguelcsx !

**Comment by rdyro** [comment]
> Hi, if this issue isn't closed can i work on it?

Definitely! Let me know once you make a PR, thanks!

**Comment by rni418** [comment]
Filed an internal issue about the CLA. Hang tight.

**Comment by IronwallxR5** [comment]
Hi @fabianp , I'm going to start working on this issue, could you assign it to me?

**Comment by fabianp** [comment]
@IronwallxR5 : I think @shreyans413 was already working on it. Keep us posted @shreyans413 😄

**Comment by shreyans413** [comment]
Yes, I was working on this, and I’ve submitted a PR with the updated docstring, explanation, and example. Let me know if any further improvements are needed! :)

**Comment by shreyans413** [comment]
I have already signed the CLA, still its failing the CLA check..

**Comment by rdyro** [comment]
@shreyans413 
> I have already signed the CLA, still its failing the CLA check..

Could you force push and try again now?

**Comment by shreyans413** [comment]
I’ve force-pushed the changes, and the CLA issue is now resolved. Let me know if anything else is needed. Thanks!

**Comment by vroulet** [comment]
Done in #1220
